Sources/web3swift/Tokens/ERC820/Web3+ERC820.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Core
 
 // Pseudo-introspection using a registry contract
```

Comment:
Please, change to documentation and add this link https://eips.ethereum.org/EIPS/eip-820